src/app/progress/progress.scss

```diff
@@ -3,7 +3,10 @@
 .progress-page {
   margin: 16px auto !important;
   user-select: text;
-
+  max-width: 90%;
+  @media (max-width: 540px) {
```

Comment:
Use `@include phone-portrait` instead, it'll use our standard media query.

Reply:
switched :D